master/internal/api_experiment.go

```diff
@@ -2100,11 +2100,16 @@ func sortExperiments(sortString *string, experimentQuery *bun.SelectQuery) error
 				"."+metricQualifier)
 			if !slices.Contains(SummaryMetricStatistics, metricQualifier) {
 				return status.Errorf(codes.InvalidArgument,
-					"sort training metrics by statistic: count, last, max, min, or sum")
+					"sort training metrics by statistic: last, max, min, or mean")
+			}
+			if metricQualifier == "mean" { //nolint: goconst
+				locator := bun.Safe(fmt.Sprintf("trials.summary_metrics->'avg_metrics'->'%s'", metricName))
+				experimentQuery.OrderExpr("(?0->>'sum')::float8 / (?0->>'count')::int ?1",
```

Comment:
**question:** is there a need to check or handle if `count` is 0?

Reply:
I don't think count can be 0.